index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">    
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar" class="nav">
+      <ul class="nav-list">
+        <li><a href=#welcome-section>About</a></li>
+        <li><a href=#projects>Work</a></li>
+        <li><a href=#contact>Contact</a></li>
+      </ul>      
+    </nav>
+
+    <section id="welcome-section"><h1>Hello World! 
+      <br> Welcome to Angom & Mark's Space
+      <br> We are software developers!</h1>
+    </section>
+
+    <section id="projects">
+      **<h1>**Ours Projects</h2>
```

Comment:
```suggestion
      **<h1>**Ours Projects</h2>
```
I would like to suggest you change h1 to h2. The opening tag should match the closing tag. Having more than one h1 element is not very semantic.